.github/workflows/ci.yaml

```
@@ -39,6 +39,33 @@ jobs:
           # Setting LD_LIBRARY_PATH is a workaround for https://github.com/TheBevyFlock/bevy_new_2d/pull/318#issuecomment-2585935350
           LD_LIBRARY_PATH="$(rustc --print target-libdir)" cargo test --locked --workspace --all-features --doc
 
+  # Check that the game builds for Wasm.
+  build-wasm:
+    name: Build for Wasm
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Install dependencies
+        run: sudo apt-get update; sudo apt-get install --no-install-recommends libasound2-dev libudev-dev libwayland-dev libxkbcommon-dev
+
+      - name: Populate target directory from cache
+        uses: Leafwing-Studios/cargo-cache@v2
+        with:
+          sweep-cache: true
+
+      - name: Install Bevy CLI
+        run: cargo install --git https://github.com/TheBevyFlock/bevy_cli --locked bevy_cli
+
+      - name: Build for Wasm
+        run: |
+          bevy build --locked --workspace --all-targets web
```

```suggestion
          bevy build --locked --workspace --all-targets --yes web
```
The release workflow includes `--yes`, not sure if we need it here as well. Also wondering what `--workspace --all-targets` does?

---

`--all-targets` is short for `--docs --examples --benchmarks`, `--workspace` does... something? Don't know, just copied it from the other definitions

---

Ok, `--all-targets` is a confusing name considering "targets" also means target platforms SMH